django_simple_bulma/utils.py

```diff
@@ -14,19 +14,39 @@
 
 # If BULMA_SETTINGS has not been declared if no extensions
 # have been defined, default to all extensions.
-if hasattr(settings, "BULMA_SETTINGS"):
-    extensions = settings.BULMA_SETTINGS.get("extensions", [])
-    fontawesome_token = settings.BULMA_SETTINGS.get("fontawesome_token", "")
-    for key in settings.BULMA_SETTINGS:
-        match = variables_name_re.match(key)
-        if match:
-            themes.append(match.group("name"))
-else:
+try:
+    if hasattr(settings, "BULMA_SETTINGS"):
+        extensions = settings.BULMA_SETTINGS.get("extensions", [])
+        fontawesome_token = settings.BULMA_SETTINGS.get("fontawesome_token", "")
+        for key in settings.BULMA_SETTINGS:
+            match = variables_name_re.match(key)
+            if match:
+                themes.append(match.group("name"))
+    else:
+        extensions = []
+        fontawesome_token = ""
+except Exception:
+    # Django settings not configured yet (e.g., during testing)
     extensions = []
     fontawesome_token = ""
 
 simple_bulma_path = Path(__file__).resolve().parent
 
+
+def get_themes() -> list:
+    """Get the list of available themes dynamically."""
+    themes = []
+    try:
+        if hasattr(settings, "BULMA_SETTINGS"):
+            for key in settings.BULMA_SETTINGS:
+                match = variables_name_re.match(key)
+                if match:
+                    themes.append(match.group("name"))
+    except Exception:
+        pass
```

Comment:
This file has a bunch of problematic and broad try except Exception's, which is an antipattern. These need to be narrowed.